{{cookiecutter.project_slug}}/setup.py

```diff
@@ -16,7 +16,17 @@
     # TODO: put package requirements here
 ]
 
+setup_requirements = [
+{%- if cookiecutter.use_pytest == 'y' %}
+    pytest-runner
```

Comment:
This doesn't look right, entries in this list need to be Python strings.

Can you please add a test for this?

I think it's just a matter of adding [a line like this](https://github.com/audreyr/cookiecutter-pypackage/blob/1d97c602eed3db2c6cfeb71882c94fc00845ef5f/tests/test_bake_project.py#L93) to [`test_using_pytest()` here](https://github.com/audreyr/cookiecutter-pypackage/blob/1d97c602eed3db2c6cfeb71882c94fc00845ef5f/tests/test_bake_project.py#L181-L186).